soc/st/stm32/common/soc_config.c

```diff
@@ -28,15 +28,16 @@ static int st_stm32_common_config(void)
 {
 #ifdef CONFIG_LOG_BACKEND_SWO
 	/* Enable SWO trace asynchronous mode */
-#if defined(CONFIG_SOC_SERIES_STM32WBX) || defined(CONFIG_SOC_SERIES_STM32H5X)
+#if defined(CONFIG_SOC_SERIES_STM32WBX) || defined(CONFIG_SOC_SERIES_STM32H5X) ||                  \
+	defined(CONFIG_SOC_SERIES_STM32H7RSX) || defined(CONFIG_SOC_SERIES_STM32U5X) ||            \
+	defined(CONFIG_SOC_SERIES_STM32L5X)
```

Comment:
Nit: it would be better to have the series by alphabetical order (and maybe one per line for easier potential additions later)

Reply:
changed to alphanum order